docs/sdk/teamsfx.querywithtoken.md

```diff
@@ -0,0 +1,43 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@microsoft/teamsfx](./teamsfx.md) &gt; [queryWithToken](./teamsfx.querywithtoken.md)
+
+## queryWithToken() function
+
+Users execute query with SSO or Access Token.
+
```

Comment:
Maybe can change to `Users execute query in message extension with SSO or access token.`

Reply:
thanks, updated.